filters/ThirdParty/filter_230_FanboyEspanol/template.txt

```diff
@@ -1,2 +1,2 @@
 !
-@include "http://fanboy.co.nz/fanboy-espanol.txt" /exclude="../../exclusions.txt"
\ No newline at end of file
+@include "https://www.fanboy.co.nz/fanboy-espanol.txt" /exclude="../../exclusions.txt"
```

Comment:
`https://fanboy.co.nz/fanboy-espanol.txt` is correct

Reply:
Noted, accepted, and changed.